bytestring/common/src/Hex.kt

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright 2010-2023 JetBrains s.r.o. and Kotlin Programming Language contributors.
+ * Use of this source code is governed by the Apache 2.0 license that can be found in the LICENSE.txt file.
+ */
+
+@file:OptIn(ExperimentalStdlibApi::class)
+
+package kotlinx.io.bytestring
+
+/**
+ * Formats bytes in this byte string using the specified [format].
+ *
+ * Note that only [HexFormat.upperCase] and [HexFormat.BytesHexFormat] affect formatting.
```

Comment:
Should it be `bytes` instead of `BytesHexFormat`?

Reply:
I'm not sure. I did so for consistency with the stdlib docs